packages/lib/calendar/config.ts

```diff
@@ -0,0 +1,13 @@
+import type { App } from "@calcom/types/App";
+
+export const APPS = {
+  caldav_calendar: {
```

Comment:
Any brave devs wanting to migrate this last one to App Store?

Reply:
It seems similar to MS Exchange so I can take a stab at this after I'm done